modules/core/src/main/scala/org/tessellation/domain/trust/storage/TrustStorage.scala

```diff
@@ -5,6 +5,7 @@ import org.tessellation.schema.trust.{InternalTrustUpdateBatch, PublicTrust, Tru
 
 trait TrustStorage[F[_]] {
   def updateTrust(trustUpdates: InternalTrustUpdateBatch): F[Unit]
+  def updateTrust(trustUpdates: Option[InternalTrustUpdateBatch]): F[Unit]
```

Comment:
There is no need for such function. You should fold the option and execute update only for some

Reply:
Changed.